libs/api/domains/driving-license/src/lib/drivingLicense.service.ts

```diff
@@ -191,10 +245,12 @@ export class DrivingLicenseService {
     studentNationalId: string,
     teacherNationalId: User['nationalId'],
   ): Promise<NewDrivingAssessmentResult> {
-    await this.drivingLicenseApi.newDrivingAssessment({
-      kennitala: studentNationalId,
-      kennitalaOkukennara: teacherNationalId,
-      dagsetningMats: new Date(),
+    await this.drivingLicenseApi.apiOkuskirteiniNewDrivingassesmentPost({
```

Comment:
Should assessment be lower case ?

Reply:
generated client.. not really for me to say